app/schemas/config.py

```diff
@@ -31,10 +31,21 @@ class SpamSchema(Schema):
     allowUrls = fields.Boolean(default=False)
 
 
+class WhitelistSchema(Schema):
+    whitelisted_urls = fields.List(fields.String())
```

Comment:
`whitelisted_urls = fields.List(fields.String(), default=[])`

Same for `_words` and backlists